### PR TITLE
update dependencies and tests

### PR DESCRIPTION
**Pull request by nmn**
Here's my try to update this package to support luxon v2. I'm not super familiar with the codebase, so please review carefully!

### CODE REVIEW

**Comment by jgclark** [comment]
Thanks, @nmn. It would be great to get this quickly aligned with luxon v2.

**Comment by skwasha** [comment]
@amaidah Any chance of this one getting merged anytime soon? Or better yet, an updated one to handle Luxon 3.x.

**Comment by nmn** [comment]
@skwasha looking at the changes introduced in Luxon 3.0, there's only a single breaking change that makes an option work differently without changing the API. So, `luxon-business-days` *should* work with luxon 3 without any changes.

**Comment by amaidah** [comment]
Hi @nmn, appreciate you taking a stab. I updated `luxon-business-days` all the way to luxon 3 here: https://github.com/amaidah/luxon-business-days/releases/tag/v3.0.1.

File: src/index.test.js [hunk 1]
```diff
@@ -296,14 +296,16 @@ describe('time zone is carried over after a business-day operation', () => {
...
+
+    const utc = DateTime.fromObject(
+      {
+        year: 2020,
+        month: 12,
+        day: 6,
+      },
+      { zone: 'utc' }
+    );
     const utcPlusTen = utc.plusBusiness({ dats: 10 });
```
Comment (amaidah) [inline src/index.test.js:309]: fixed `dats`: https://github.com/amaidah/luxon-business-days/pull/72/files#diff-38a2b85ac1b61b44eb1945e128241cb5ef01e5646cf06b586f5c0114b153fa0eR309

**Pull request closed without merging**